components/BottomSheet/src/MDCBottomSheetController.m

```diff
@@ -38,6 +40,7 @@ - (instancetype)init {
   self = [super init];
   if (self) {
     _elevation = MDCShadowElevationModalBottomSheet;
+    _mdc_overrideBaseElevation = -1;
```

Comment:
is this init ever called?

Reply:
It is only called in our tests but because we can't measure who uses what via CocoaPods we still need to support this since it isn't marked unavailable. I've added a deprecation message to `init`.